src/components/switch/switch.component.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useCallback } from "react";
+import { MarginProps } from "styled-system";
+
+import SwitchStyle from "./switch.style";
+import CheckableInput, {
+  CommonCheckableInputProps,
+} from "../../__internal__/checkable-input";
+import SwitchSlider from "./__internal__/switch-slider.component";
+import useIsAboveBreakpoint from "../../hooks/__internal__/useIsAboveBreakpoint";
+import { TooltipProvider } from "../../__internal__/tooltip-provider";
+import { filterStyledSystemMarginProps } from "../../style/utils";
+
+export type LabelAlign = "left" | "right";
```

Comment:
I'm not sure if there's a good reason to export the `LabelAlign` type or not (although I personally don't see one). But if there is, should it not also be exported from `index.ts`?